loaf-explainer.md

```diff
@@ -346,6 +346,15 @@ instead? Are those metrics useful?
 minimal overhead, while profiling exposes "everything" with some more overhead, but the differences
 and relationship can be further understood.
 
+## Relationship with TBT
+
+TBT ([total blocking time](https://developer.chrome.com/docs/lighthouse/performance/lighthouse-total-blocking-time/)) is a [lighthouse](https://developer.chrome.com/docs/lighthouse/overview/) metric that allows measuring how responsive/sluggish the experience is during page load.
```

Comment:
https://web.dev/tbt/ may be a more generic link. The metric is not LH specific, as it is used in other lab tools. It is also collected in the field by some RUM providers.